.vscode/launch.json

```diff
@@ -35,6 +35,27 @@
       "console": "integratedTerminal",
       "internalConsoleOptions": "neverOpen"
     },
+    {
+      "name": "(PNH) Debug Current Automation Test",
```

Comment:
We normally don't check in launch configs or tasks which are user specific.  I would suggest changing the launch config a little and make it user-agnostic...
```
 {
      "name": "Debug Current Performance Test",
      "type": "node",
      "request": "launch",
      "program": "${workspaceRoot}/node_modules/@wdio/cli/bin/wdio.js",
      "args": [
        "run",
        "test/wdio.conf.ts",
        "--spec",
        "${file}"
      ],
      "cwd": "${workspaceFolder}",
      "autoAttachChildProcesses": true,
      "console": "integratedTerminal",
      "internalConsoleOptions": "neverOpen"
    },
```

...and then also set the necessary environment variables:
```
(in a terminal window)
$ export EXTENSION_PATH="/Users/peter.hale/git/vse/packages"
$ export DEV_HUB_SFDX_AUTH_URL="force://PlatformCLI::5Aep8612E5JLxJp3ESIXoO277mV4I.qJxA4bxutk8bsnO5B93z3sW1ibxM3WkNhN.lnmlQw6qDjUkIUjUtZ4BC1@na40-dev-hub.my.salesforce.com"
$ export DEV_HUB_USER_NAME="admin@integrationtesthubna40.org"      
```

And if it were me, I'd create a shell script with those three exports in them, and run the script anytime I needed the env vars set.